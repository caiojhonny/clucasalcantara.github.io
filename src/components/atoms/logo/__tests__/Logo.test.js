import React from 'react'
import { shallow } from 'enzyme'
import Logo from '..'

describe('Logo component', () => {
  it('should renders without crash', () => {
    const wrapper = shallow(<Logo content="Work with coffee" />)

    expect(wrapper).toMatchSnapshot()
  })
})
