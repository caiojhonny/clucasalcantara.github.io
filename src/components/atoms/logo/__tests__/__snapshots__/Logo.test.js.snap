// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component should renders without crash 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Logo
    color="black"
    content="Work with coffee"
    subtitle="Sua logo aqui"
    subtitleColor="blue"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <div
          className="content"
        >
          Work with coffee
        </div>,
        <ForwardRef(render)
          color="blue"
        >
          Sua logo aqui
        </ForwardRef(render)>,
      ],
      "color": "black",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Work with coffee",
          "className": "content",
        },
        "ref": null,
        "rendered": "Work with coffee",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "Sua logo aqui",
          "color": "blue",
        },
        "ref": null,
        "rendered": "Sua logo aqui",
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "__emotion_base": "div",
          "__emotion_forwardProp": undefined,
          "__emotion_real": [Circular],
          "__emotion_styles": Array [
            [Function],
          ],
          "defaultProps": undefined,
          "displayName": "Styled(div)",
          "render": [Function],
          "withComponent": [Function],
        },
      },
    ],
    "type": Object {
      "$$typeof": Symbol(react.forward_ref),
      "__emotion_base": "div",
      "__emotion_forwardProp": undefined,
      "__emotion_real": [Circular],
      "__emotion_styles": Array [
        [Function],
      ],
      "defaultProps": undefined,
      "displayName": "Styled(div)",
      "render": [Function],
      "withComponent": [Function],
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <div
            className="content"
          >
            Work with coffee
          </div>,
          <ForwardRef(render)
            color="blue"
          >
            Sua logo aqui
          </ForwardRef(render)>,
        ],
        "color": "black",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Work with coffee",
            "className": "content",
          },
          "ref": null,
          "rendered": "Work with coffee",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "Sua logo aqui",
            "color": "blue",
          },
          "ref": null,
          "rendered": "Sua logo aqui",
          "type": Object {
            "$$typeof": Symbol(react.forward_ref),
            "__emotion_base": "div",
            "__emotion_forwardProp": undefined,
            "__emotion_real": [Circular],
            "__emotion_styles": Array [
              [Function],
            ],
            "defaultProps": undefined,
            "displayName": "Styled(div)",
            "render": [Function],
            "withComponent": [Function],
          },
        },
      ],
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "__emotion_base": "div",
        "__emotion_forwardProp": undefined,
        "__emotion_real": [Circular],
        "__emotion_styles": Array [
          [Function],
        ],
        "defaultProps": undefined,
        "displayName": "Styled(div)",
        "render": [Function],
        "withComponent": [Function],
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
